data_collection/web/guides.rst

```diff
@@ -221,187 +221,184 @@ indicate no value.
-    _paq.push(["addEcommerceItem", "82775027", "MEGA Milk Chocolate 200g", "Candy", 6.00, 1]);
-    _paq.push(["trackEcommerceCartUpdate", 6.00]);
+  * cart update (:ref:`ecommerceCartUpdate<jtc-api-ecommerceCartUpdate>`)
+  * product addition and removal (:ref:`ecommerceAddToCart<jtc-api-ecommerceAddToCart>` and :ref:`ecommerceRemoveFromCart<jtc-api-ecommerceRemoveFromCart>`)
 
-    // visitor added one mango fruit to a shopping cart with one chocolate bar
-    _paq.push(["clearEcommerceCart"]);
-    _paq.push(["addEcommerceItem", "82775027", "MEGA Milk Chocolate 200g", "Candy", 6.00, 1]);
-    _paq.push(["addEcommerceItem", "01809926", "FRUTASTIC Mango", "Fruits & vegetables", 4.00, 1]);
-    _paq.push(["trackEcommerceCartUpdate", 10.00]);
+Cart update allows us to synchronize longer living cart session (containing previously selected products) with short living visitor session.
+This command is optional but recommended. It should be used once per page (immidiately after loading).
 
-    // visitor removed one chocolate from a shopping cart with one chocolate bar and one mango
-    _paq.push(["clearEcommerceCart"]);
-    _paq.push(["addEcommerceItem", "01809926", "FRUTASTIC Mango", "Fruits & vegetables", 4.00, 1]);
-    _paq.push(["trackEcommerceCartUpdate", 4.00]);
-
-2. Replicate visitor's interactions with the cart using functions :ref:`addEcommerceItem<jtc-api-addEcommerceItem>`, :ref:`removeEcommerceItem<jtc-api-addEcommerceItem>`, :ref:`clearEcommerceCart<jtc-api-clearEcommerceCart>`.
+Example:
 
 .. code-block:: javascript
 
-    // visitor added one chocolate bar to an empty shopping cart
-    _paq.push(["addEcommerceItem", "82775027", "MEGA Milk Chocolate 200g", "Candy", 6.00, 1]);
-    _paq.push(["trackEcommerceCartUpdate", 6.00]);
-
-    // visitor added one mango fruit to a shopping cart with one chocolate bar
-    _paq.push(["addEcommerceItem", "01809926", "FRUTASTIC Mango", "Fruits & vegetables", 4.00, 1]);
-    _paq.push(["trackEcommerceCartUpdate", 10.00]);
-
-    // visitor removed one chocolate bar from a shopping cart with one chocolate bar and one mango
-    _paq.push(["removeEcommerceItem", "82775027"]);
-    _paq.push(["trackEcommerceCartUpdate", 4.00]);
+  _paq.push([
+      "ecommerceCartUpdate",
+      [
+          {
+              sku: "craft-311",
+              name: "Unicorn Iron on Patch",
+              category: "Crafts & Sewing",
+              price: "50.00",
+              quantity: 3,
+              brand: "DMZ",
+              variant: "blue"
+          }
+          {
+              sku: "craft-312",
+              name: "Unicorn Iron on Grass",
+              category: "Crafts & Sewing",
+              price: "30.00",
+              quantity: 1,
+              brand: "DMZ",
+              variant: "red"
+          }
+      ],
+      "180.00"
+  ]);
+
+
+Product addition and removal commands are used immidiately when visitor adds to a cart or removes from a cart any product.
```

Comment:
Same as above, `immediately`. Also, remember about "a/an" articles - it should be `when a visitor` not `when visitor`.

Reply:
```suggestion
Product addition and removal commands are used immediately when a visitor adds to a cart or removes from a cart any product.
```